hack/jenkins/job-configs/kubernetes-e2e.yaml

```diff
@@ -358,13 +358,10 @@
             test-owner: 'beeps'
             provider-env: '{gke-provider-env}'
             job-env: |
-                # XXX Not a unique project
                 export E2E_NAME="e2e-gke-ingress"
```

Comment:
You can get rid of `E2E_NAME` since this is the only thing running in this project; and I believe `FAIL_ON_GCP_RESOURCE_LEAK` is true by default.